packages/controllers/src/controllers/OptionsStateController.ts

```diff
@@ -0,0 +1,34 @@
+import { proxy, subscribe as sub } from 'valtio/vanilla'
```

Comment:
How do you persist the state? Like if they click accept and revisit the page later?

Reply:
Should we ask again them to accept?